widget/embedded/src/components/WalletModal/WalletModal.types.ts

```diff
@@ -3,9 +3,9 @@ import type { WalletState } from '@rango-dev/ui';
 export interface PropTypes {
   open: boolean;
   onClose: () => void;
-  image: string;
-  state: WalletState;
-  error?: boolean;
+  image?: string;
+  state?: WalletState;
```

Comment:
It should alway has a state, why did you make it optional?

Reply:
fixed.